src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -128,7 +128,7 @@ export default {
             body: 'A new version of this app is available!',
             delay: 0,
             onClick: () => {
-                App.triggerUpdateAvailable();
+                Update.triggerUpdateAvailable();
```

Comment:
I moved `triggerUpdateAvailable` into a separate `Update` action to avoid a dependency cycle between `App > Report > LocalNotification > BrowserNotifications > App`.